testnets/atomonetestnet1/assetlist.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "atomonetestnet1",
```

Comment:
let's use the non-numbered chain_name fist, and then use the number suffixes for additional or archived testnets